samples/cellular/udp/README.rst

```diff
@@ -7,7 +7,7 @@ Cellular: UDP
    :local:
    :depth: 2
 
-The UDP sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
+The Cellular: UDP sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
```

Comment:
```suggestion
This sample demonstrates the sequential transmission of UDP packets to a predetermined server identified by an IP address and a port.
```

Reply:
The reason to explicitly mention the sample name as cellular: UDP is, we have a similar sample in the networking folder which has Wi-Fi support. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/net/udp/README.html.

So, I think we could keep this as is?

Reply:
I'd say we can still safely say "This sample", as the reader is possibly on this page when reading.

Reply:
Discussed offline, and decided to keep the sample name at the first instance.